certified-connectors/Leadinfo/apiDefinition.swagger.json

```diff
@@ -0,0 +1,293 @@
+            },
+            "branch_local_01_name": {
+              "type": "string",
+              "description": "branch_local_01_name"
+            },
+            "branch_international_01_name": {
+              "type": "string",
+              "description": "branch_international_01_name"
+            },
+            "linkedin": {
+              "type": "string",
+              "description": "linkedin"
+            },
+            "xing": {
+              "type": "string",
+              "description": "xing"
+            },
+            "leadinfo_link": {
+              "type": "string",
+              "description": "leadinfo_link"
+            },
+            "timestamp": {
+              "type": "string",
+              "description": "timestamp"
+            },
+            "page_url_first_visit": {
+              "type": "string",
+              "description": "page_url_first_visit"
+            },
+            "branch_code": {
+              "type": "string",
+              "description": "branch_code"
+            },
+            "branch_code_international": {
+              "type": "string",
+              "description": "branch_code_international"
+            }
+          }
+        },
+        "description": "webhook response"
+      },
+      "put": {
+        "summary": "Triggers when a company is send",
+        "description": "Triggers when a company is send",
+        "responses": {
+          "201": {
+            "description": "success"
+          }
+        },
+        "operationId": "Webhook-company-trigger",
+        "x-ms-trigger": "single",
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "required": false,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "url": {
+                  "type": "string",
+                  "description": "url",
+                  "x-ms-notification-url": true,
+                  "x-ms-visibility": "internal",
+                  "title": ""
+                },
+                "integration": {
+                  "type": "string",
+                  "description": "integration",
+                  "title": "",
+                  "x-ms-visibility": "internal",
+                  "default": "microsoftpowerautomate"
+                },
+                "type": {
+                  "type": "string",
+                  "description": "Flow type",
+                  "title": "Type",
+                  "default": "share",
+                  "enum": [
+                    "share",
+                    "trigger"
+                  ]
+                }
+              },
+              "required": [
+                "url",
+                "integration",
+                "type"
+              ]
+            }
+          }
+        ]
+      }
+    },
+    "/v1/webhook/{id}": {
+      "delete": {
+        "responses": {
+          "default": {
```

Comment:
@GerbenZ01 it is not allowed to have a default response only. is it possible to define a proper response. Please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/create-webhook-trigger#creating-the-webhook).